components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarAutocompleteFeature.kt

```diff
@@ -21,25 +21,30 @@ import mozilla.components.concept.toolbar.Toolbar
  */
 class ToolbarAutocompleteFeature(
     val toolbar: Toolbar,
-    val engine: Engine? = null
+    val engine: Engine? = null,
+    val shouldShowResult: () -> Boolean = { true }
```

Comment:
This should probably be something closer to `shouldAutocomplete` with an appropriate property description. I would even go as far as making the commit message/title clearer since autocomplete results could be implied as suggestion results. So maybe mention that we are adding an option to disable autocompletion results in the  awesomebar input.

Reply:
Agreed.  I'll update.